org.kuchelmeister.ToolboxTuner.json

```diff
@@ -0,0 +1,59 @@
+{
+    "id" : "org.kuchelmeister.ToolboxTuner",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "46",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable",
+        "org.freedesktop.Sdk.Extension.llvm15"
```

Comment:
are you sure you need such old version of llvm? latest one is llvm18

Reply:
Good catch! Bumped the version to llvm18 and ran a testbuild locally.